src/clone-node.ts

```diff
@@ -37,25 +37,19 @@ function cloneVideoElement(video: HTMLVideoElement) {
   }
 }
 
-async function cloneIFrameElement(iframe: HTMLIFrameElement) {
+function cloneIFrameElement(iframe: HTMLIFrameElement) {
   try {
     if (iframe?.contentDocument?.body) {
-      return (await cloneNode(
-        iframe.contentDocument.body,
-        {},
-      )) as HTMLBodyElement
+      return cloneNode(iframe.contentDocument.body, {}) as HTMLBodyElement
     }
   } catch {
-    // Failed to clone iframe
+    console.error('Failed to clone iframe', iframe)
```

Comment:
I am commenting once but it applies to _all_ your PRs. Do not `console.error` without some prefix that tells us that it came from inScreen. You're making customer sites' logs ugly, at least tell them it came from us and not from their code.